source/config-file/config-file-options.txt

```diff
@@ -59,6 +59,21 @@ You can configure the following settings in a configuration file:
      - Configure |compass| to not perform outgoing network operations other 
        than those to the database.
 
+Example 
+-------
+To view a sample |compass-short| configuration file, run the following command: 
+
+.. code-block:: sh
+
+     <path-to-Compass-executable> --show-example-config
+
```

Comment:
I don't think that command will run compass with the sample config. It looks like it is creating a sample config file using the shell's redirect operator ">" to make a copy of the sample file. 

If that's right, consider something like: 

You can use the following command to create a config file based on the sample configuration file.

As the line is currently written, if compass only reads it's config once, it won't use the new file the first time you run the command. The 2d time you run it, it will overwrite any modifications you might have made and may or may not read it first (I'm not sure of the operator precidence). You can test that by running once, adding a comment to the file, running a command the second time and checking what happens. Most likely they comment will be gone after the 2d time around.